dump_zenodo_open_metadata.py

```diff
@@ -53,53 +53,65 @@ def parse_record(rec, spam=False):
     out["spam"] = spam
     return out
 
+print("What type of dump you want? 1- Ham, 2- Spam or 3- Both?")
+type = input()
+
+if(type == 1):
+    type_string = "ham_"
+elif(type == 2):
+    type_string = "spam_"
+else:
+    type_string = "ham&spam_"
+
 today = datetime.date.today().isoformat()
-FILENAME = f"zenodo_open_metadata_{today}.jsonl.gz"
+FILENAME = "zenodo_open_metadata_" + type_string + today + ".jsonl.gz"
 
-records_ham = (
-    db.session.query(RecordMetadata)
-    .join(PersistentIdentifier, PersistentIdentifier.object_uuid == RecordMetadata.id)
-    .filter(
-        PersistentIdentifier.pid_type == "recid",
-        PersistentIdentifier.status == "R",
-        PersistentIdentifier.object_type == "rec",
+if(type != 2):
```

Comment:
```suggestion
if dump_type in ('spam', 'both'):
```

- In this case, checking against the `spam` value is more friendly to read (vs. `1` which one has to check what value it is for)
- If we want `both` that also works :)
- In general, one should **never** need for parentheses around `if`s. If you end up with a very complex boolean check, it's better to break the logic down to separate variables before the `if` statement